modules/enabling-team-sync-oidc.adoc

```diff
@@ -42,8 +43,9 @@ FEATURE_UI_V2: true
 <4> Required. The name of the service that is being authenticated.
 <5> Required. The key name within the OIDC token payload that holds information about the user's group memberships. This field allows the authentication system to extract group membership information from the OIDC token so that it can be used with {productname}.
 <6> Required. Adds additional scopes that {productname} uses to communicate with the OIDC provider. Must include `'openid'`. Additional scopes are optional.
-<7> Required. Whether to allow for team membership to be synced from a backing group in the authentication engine.
-<8> Optional. If enabled, non-superusers can setup team synchronization.
+<7> Whether to allow or disable the `user_endpoint` returned for Azure Entra ID OIDC providers. Defaults to `false`.
```

Comment:
It is not just for Azure. But, for Azure we want this to be `true` because Azure suggest getting the user's information from the token instead of calling the UserInfo endpoint. And user groups are present in the token and not in the userinfo endpoint.